ecpp/average.ecpp

```diff
@@ -322,7 +341,12 @@ bool database_ready;
                         char *value = zmsg_popstr (recv_msg);
 
                         if (!got_timestamps) {
-                            csv_data [0].push_back (timestamp);
+                            time_t rawtime = atoi(timestamp);
+                            struct tm timeinfo;
+                            localtime_r(&rawtime, &timeinfo);
```

Comment:
it seems that localtime_r() (POSIX) is not thread-safe... 
you should use strftime() instead.

Reply:
They don't seem to be alternatives. localtime converts seconds from epoch(time_t) to tm struct. even if I use strftime I need to do conversion.